test/e2e/framework.go

```diff
@@ -1957,8 +1957,10 @@ func (data *TestData) RunCommandFromPod(podNamespace string, podName string, con
 	if err != nil {
 		return "", "", err
 	}
+	ctx, cancelFn := context.WithTimeout(context.Background(), 1*time.Minute)
```

Comment:
does that 1 minute timeout match the previous behavior of `Stream`?

Reply:
I think so. The context is just for avoiding resource leak for some bad streams which will hang there forever.

Reply:
Ok, let's go with this then. We can add a `context.Context` parameter to the function later if needed